src/settings/SettingsPage.js

```diff
@@ -63,15 +64,21 @@ export const SettingsPage = ({
         </div>
       </Box> }
 
-    <Box className='mb3 pa4-l pa2'>
-      <div className='lh-copy charcoal'>
+      <Box className='mb3 pa4-l pa2'>
+        <div className='lh-copy charcoal'>
```

Comment:
the indentation level here should be where the opening brace on line 55 is.

```suggestion
    <Box className='mb3 pa4-l pa2'>
      <div className='lh-copy charcoal'>
```

but also, we should have lint/style tools handle this automatically for us